ocular/src/chain/client/query.rs

```diff
@@ -1,9 +1,145 @@
-use crate::error::{ChainClientError, RpcError};
+use crate::{
+    cosmos_modules::{base::Coin, *},
+    error::{ChainClientError, GrpcError, RpcError},
+};
 use tendermint_rpc::Client;
+use tonic::transport::Channel;
 
 use super::ChainClient;
 
+pub type AuthQueryClient = auth::query_client::QueryClient<Channel>;
+pub type BankQueryClient = bank::query_client::QueryClient<Channel>;
+pub type DistributionQueryClient = distribution::query_client::QueryClient<Channel>;
+pub type EvidenceQueryClient = evidence::query_client::QueryClient<Channel>;
+pub type GovQueryClient = gov::query_client::QueryClient<Channel>;
+pub type MintQueryClient = mint::query_client::QueryClient<Channel>;
+pub type ParamsQueryClient = params::query_client::QueryClient<Channel>;
+pub type SlashingQueryClient = slashing::query_client::QueryClient<Channel>;
+pub type StakingQueryClient = staking::query_client::QueryClient<Channel>;
```

Comment:
We could definitely do that as we implement them. The important thing to me was that it extends the `ChainClient` impl block so there is no extra module importing for these convenience methods, and that whatever structure we come up with is re-exported to make the public facing modules clean. i.e. if a user wants a `GovQueryClient` we should re-export that to something like `ocular::query::GovQueryClient` rather than `ocular::chain::client::query::gov::GovQueryClient`